apps/studio/src/plugins/ProductTourPlugin.ts

```diff
@@ -0,0 +1,130 @@
+import { PluginObject } from "vue";
+import { driver as createDriver, DriveStep } from "driver.js";
+import { UtilityConnection } from "@/lib/utility/UtilityConnection";
+import type store from "@/store";
+
+type Context = {
+  store: typeof store;
+  utility: UtilityConnection;
+};
+
+type FlowId = "connectedScreen";
+
+type FlowStep = DriveStep & {
+  shouldShow: (context: Context) => boolean | Promise<boolean>;
+  onFinished: (context: Context) => void | Promise<void>;
+};
+
+const flows: Record<
+  FlowId,
+  {
+    steps: FlowStep[];
+  }
+> = {
+  connectedScreen: {
+    steps: [
+      {
+        element: "#add-tab-group",
+        popover: {
+          title: `<div class="main-title"><i class="material-icons ai-shell-icon">auto_awesome</i> AI Shell</div><div class="subtitle">Included in your paid plan</div>`,
+          description: `Use <b>AI Shell</b> from this menu to ask questions and generate queries.`,
+          side: "bottom",
+          showButtons: ["next"],
+          doneBtnText: "Okay",
+        },
+        async shouldShow(context: Context) {
+          if (context.store.getters.isCommunity) {
+            return false;
+          }
+
+          const settings = context.store.getters["settings/settings"];
+          if (settings["tabDropdownAIShellHintShown"]) {
+            return false;
+          }
+
+          // Dont show it if the plugin is not in the dropdown menu (disabled or not installed)
+          if (
+            !context.store.getters["tabs/newTabDropdownItems"].find(
+              (t) => t.config.pluginId === "bks-ai-shell"
+            )
+          ) {
+            return false;
+          }
+
+          const tabs = await context.utility.send("appdb/tabs/find", {
```

Comment:
i think here we only care if they've EVER used the ai shell right? Do we care what workspace they used it in?

Like if I use a lot in dev, I don't need reminding in production. I know it exists.